### ISSUE TITLE
Random space in editor

### ISSUE THREAD

**Issue by erictleung** [FreeCodeCamp/FreeCodeCamp#5688]
Challenge [Waypoint: Storing Values with the Equal Operator](http://www.freecodecamp.com/challenges/waypoint-storing-values-with-the-equal-operator#?solution=%2F%2F%20Setup%0Avar%20a%3B%0Avar%20b%20%3D%202%3B%0A%0A%2F%2F%20Only%20change%20code%20below%20this%20line%0A%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.106 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript
// Setup
var a;
var b = 2;

// Only change code below this line


```

This is a very minor issue, almost trivial, but this Waypoint initializes the editor with an unnecessary space right under `// Only change code below this line`.

The space can be found if you look back at the code https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/seed/challenges/01-front-end-development-certification/basic-javascript.json#L120. It should be `""` instead of `" "`.

**Comment by ShaggyTech** [comment]
Easy enough.  I'll grab this one.

**Comment by ShaggyTech** [comment]
So maybe I'm doing something wrong but I just fetched the latest from "staging" and there is no space between the quotation marks in the latest version.  This appears to have already been corrected.

**Comment by ltegman** [comment]
Hmmm, I'm not sure why you're not seeing it on your end. I still see it on github and in the most recent staging I just pulled down.

**Comment by ShaggyTech** [comment]
My mistake, I hadn't pushed to my forked repo and I guess that was keeping the latest fetched files from showing.  Should be fixed with #5692
